### DISCUSSION TITLE
PaoPaoGateWay搭建有问

### DISCUSSION THREAD

**Discussion by dawangLM** [kkkgo/PaoPaoGateWay#59] [问和答]
pve使用docker定制ISO镜像，倒入后开机报错80端口拒绝连接，请问是什么错误

![1](https://github.com/kkkgo/PaoPaoGateWay/assets/62083063/39cab550-b9d9-4454-8513-8f21c8c7f89d)

**Comment by kkkgo** [comment]
你应该往前看日志，这是clash内核都没起来。

**Reply by dawangLM** [reply]
![截屏2024-02-08 19 42 51](https://github.com/kkkgo/PaoPaoGateWay/assets/62083063/bfc3fa07-f844-41e7-bf73-586022b60352)
看样子好像是内核的不符合的问题

**Reply by dawangLM** [reply]
但是clash内核确实没选错›
![截屏2024-02-08 19 45 38](https://github.com/kkkgo/PaoPaoGateWay/assets/62083063/437766ca-2b0d-479e-a3da-e57c6f6c86dc)

**Reply by kkkgo** [reply]
参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/23#discussioncomment-8390081

**Reply by dawangLM** [reply]
好的谢谢老哥

**Comment by exp4sky** [comment]
大佬们clashmeta 内核 配置了proxies 、ruleset、RuleSet 文件夹这怎么用paopaogateway，我制作了iso 但是没有把这几个文件夹加进iso里

**Reply by kkkgo** [reply]
可以部署subconverter的docker把你的配置转换成订阅url，然后ppgw直接用suburl模式。
参考 https://github.com/tindy2013/subconverter/blob/master/README-cn.md

**Reply by exp4sky** [reply]
感谢大佬
